### PR TITLE
[53505]feature profile screen

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
- [#Task 53506: Create UI](https://edu-redmine.sun-asterisk.vn/issues/53506)
- [#Task 53507: Handle Event](https://edu-redmine.sun-asterisk.vn/issues/53507)
- [#Task 53508:get data](https://edu-redmine.sun-asterisk.vn/issues/53508)
## WHAT
- create UI
- change avatar
- change profile
- change password
- logout
- get data of current user
## Evidence (Screenshot or Video)
image

## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://sal.vn/WtYPAy |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Redmine | Sun* Redmine working process  | https://sal.vn/piPoQn |<li>- [ ] yes</li>|<li>- [ ] yes</li>

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/layout/dialog_change_password.xml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="@dimen/dp_20"
+        android:paddingStart="@dimen/dp_10"
+        android:textColorHint="@color/black"
+        android:textSize="@dimen/sp_23"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/text_old_password"
+        tools:ignore="MissingConstraints" />
```
Comment (ndt0409) [inline app/src/main/res/layout/dialog_change_password.xml]: remove missing constraint nhé

File: app/src/main/res/layout/dialog_change_password.xml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="@dimen/dp_20"
+        android:paddingStart="@dimen/dp_10"
+        android:textColorHint="@color/black"
+        android:textSize="@dimen/sp_23"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/text_new_password"
+        tools:ignore="MissingConstraints" />
```
Comment (ndt0409) [inline app/src/main/res/layout/dialog_change_password.xml]: .

File: app/src/main/res/layout/dialog_change_password.xml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="@dimen/dp_20"
+        android:paddingStart="@dimen/dp_10"
+        android:textColorHint="@color/black"
+        android:textSize="@dimen/sp_23"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/text_confirm_password"
+        tools:ignore="MissingConstraints" />
```
Comment (ndt0409) [inline app/src/main/res/layout/dialog_change_password.xml]: .

File: app/src/main/res/layout/dialog_change_profile.xml [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="@dimen/dp_20"
+        android:paddingStart="@dimen/dp_10"
+        android:textColorHint="@color/black"
+        android:textSize="@dimen/sp_23"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/text_name"
+        tools:ignore="MissingConstraints" />
```
Comment (ndt0409) [inline app/src/main/res/layout/dialog_change_profile.xml]: .

File: app/src/main/res/layout/dialog_change_profile.xml [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="@dimen/dp_20"
+        android:paddingStart="@dimen/dp_10"
+        android:textColorHint="@color/black"
+        android:textSize="@dimen/sp_23"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/text_phone"
+        tools:ignore="MissingConstraints" />
```
Comment (ndt0409) [inline app/src/main/res/layout/dialog_change_profile.xml]: .

File: app/src/main/res/layout/dialog_change_profile.xml [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="@dimen/dp_20"
+        android:paddingStart="@dimen/dp_10"
+        android:textColorHint="@color/black"
+        android:textSize="@dimen/sp_23"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/text_accountname"
+        tools:ignore="MissingConstraints" />
```
Comment (ndt0409) [inline app/src/main/res/layout/dialog_change_profile.xml]: .

File: app/src/main/res/layout/dialog_change_profile.xml [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="@dimen/dp_20"
+        android:paddingStart="@dimen/dp_20"
+        android:textColorHint="@color/black"
+        android:textSize="@dimen/sp_23"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/text_accountnumber"
+        tools:ignore="MissingConstraints" />
```
Comment (ndt0409) [inline app/src/main/res/layout/dialog_change_profile.xml]: .

File: app/src/main/res/layout/fragment_profile.xml [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+        android:id="@+id/image_avatar"
+        android:layout_width="@dimen/dp_0"
+        android:layout_height="@dimen/dp_0"
+        android:layout_marginStart="@dimen/dp_20"
+        android:src="@mipmap/ic_launcher"
+        app:layout_constraintDimensionRatio="1"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@id/guideline_0.1"
+        app:layout_constraintWidth_percent="0.27"
+        tools:ignore="MissingConstraints" />
```
Comment (ndt0409) [inline app/src/main/res/layout/fragment_profile.xml]: .

File: app/src/main/res/layout/fragment_profile.xml [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+        android:id="@+id/text_name"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="10dp"
+        android:gravity="left|center"
+        android:textColorHint="@color/black"
+        android:textSize="@dimen/sp_20"
+        app:layout_constraintBottom_toTopOf="@+id/text_email"
+        app:layout_constraintStart_toEndOf="@+id/image_avatar"
+        tools:ignore="MissingConstraints" />
```
Comment (ndt0409) [inline app/src/main/res/layout/fragment_profile.xml]: .

File: app/src/main/res/layout/dialog_change_password.xml [hunk 10]
```diff
@@ -0,0 +1,114 @@
...
+    <TextView
+        android:id="@+id/text_title"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/Green"
+        android:gravity="center"
+        android:padding="@dimen/dp_10"
+        android:text="@string/changepassword"
+        android:textColor="@color/white"
+        android:textSize="30sp"
```
Comment (ndt0409) [inline app/src/main/res/layout/dialog_change_password.xml]: Nên extract sp ra file dimen nhé

**Review by minhnn-1279 — APPROVED** [review]

**Review by ndt0409 — APPROVED** [review]

**Pull request merged**